frontend/index.html

```diff
@@ -92,9 +92,6 @@
                 <button type="button" class="dropdown-toggle" data-toggle="dropdown">
```

Comment:
The menu button still remains:
![Bildschirmfoto vom 2020-12-16 12-15-31](https://user-images.githubusercontent.com/14070005/102341645-6d85db80-3f98-11eb-9d39-817c54908cf9.png)
(Deployed in Opencast)

The button does not do anything. It may be enough to just remove the whole dropdown in the index.html

Reply:
Should be fixed. :)